t/op/try.t

```diff
@@ -74,6 +74,88 @@ no warnings 'experimental::try';
     is(f(), "return inside try", 'return inside try');
 }
 
+no warnings "exiting";
+
+# eval next/last inside try
+{
+    my $result = "";
+    try {
+        {
+            $result .= "a";
+            eval "next";
+            $result .= "b";
+        }
+    } catch ($e) {
+        $result .= "c";
+    }
+
+    is($result, "a", 'eval next block inside try');
+
+    $result = "";
+    {
+        try {
+            $result .= "d";
+            eval "last";
+            $result .= "e";
+        } catch ($e) {
+            $result .= "f";
+        }
+    }
+
+    is($result, "d", 'eval last block outside try');
```

Comment:
ITYM "inside try"

Reply:
The block that the last is terminating is outside the try.

Reply:
I edited the descriptions to make them clearer